sherlock/sherlock.py

```diff
@@ -532,12 +532,12 @@ def main():
                              "On the other hand, this may cause a long delay to gather all results."
                         )
     parser.add_argument("--print-all",
-                        action="store_true", dest="print_all",
+                        action="store_true", dest="print_all", default=False,
                         help="Output sites where the username was not found."
                         )
     parser.add_argument("--print-found",
-                        action="store_false", dest="print_all", default=False,
-                        help="Output sites where the username was found."
+                        action="store_true", dest="print_found", default=False,
+                        help="Output sites where the username was found (also if exported as file)."
```

Comment:
>  (also if exported as file)
Is this needed? I feel like this would be assumed

Reply:
The current behavior is printing only those accounts that are found and writing on the CSV/XLSX all of the websites no matter if the account was found. Now, we need two variables instead of one in order to say if we want to print it all and if we want to save it all on the file. This way, the current behavior is maintained and you can personalize the printing and the writing on the file.

I kept the same name to make the minimal changes possible but it may be more intuitive to change the name of 'print_found' to 'export_all' or 'write_all'. I could do that if you think is better.

Reply:
We can keep it as it is for now. I'll think more about this and make changes if I feel if thats needed. I will make sure to credit you 👍